mitosheet/src/components/Mito.tsx

```diff
@@ -725,8 +729,18 @@ export const Mito = (props: MitoProps): JSX.Element => {
 
         const toursToDisplay: TourName[] = []
 
+        // If we are in variant B, then we display the tour to the user. In either case, we 
+        // take special care to set the user json as if we also did the other variant, so
+        // that make sure the user.json for all users is valid at the end of the experiment
```

Comment:
If we redo the user.json. Maybe this is a fragility that we want to redesign. This works for onboarding things because its one and done. But if we had something else that was ongoing, we won't want to mark the user as having completed it. 

Instead, when the experiment is done, we'll want to actually want them to have access to the X. 

On second though, this seems like the type of infrastructure design that we should definitely do with a time when we want to use it. So let's hold off on that specific change.

Reply:
I agree that we'll def want to change this, and also agree we should do it when we encounter the issue specifically!